src/kube-runtime/src/plugins/plugin_utils.py

```diff
@@ -54,3 +56,20 @@ def plugin_init():
     plugin_config = yaml.safe_load(args.plugin_config)
 
     return [plugin_config, args.pre_script, args.post_script]
+
+
+PAI_WORK_DIR = "/usr/local/pai"
+
+
+def try_to_install_by_cache(group_name: str, fallback_cmds: list):
+    source_folder = "/opt/package_cache"
+    target_folder = os.path.join(PAI_WORK_DIR, "package_cache")
+    exists_group_names = os.listdir(source_folder)
```

Comment:
pls check dir exists here to fix unit test

Reply:
Done. But the unit test should also be fixed because the commands are sightly different.